cellfinder_napari/tests/test_utils.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from imlib.cells.cells import Cell
+from qtpy.QtWidgets import QGridLayout
+
+from ..utils import add_button, add_combobox, add_layers, html_label_widget
+
+
+def test_add_layers(make_napari_viewer):
+    """Smoke test for add_layers utility"""
+    points = [
+        Cell(pos=[1, 2, 3], cell_type=Cell.CELL),
+        Cell(pos=[4, 5, 6], cell_type=Cell.UNKNOWN),
+    ]
+    viewer = make_napari_viewer()
+    add_layers(points, viewer)
```

Comment:
Would be good to add a check that `viewer.layers` is now populated with a new layer.

Reply:
Two new layers, even :)